src/Root.h

```diff
@@ -208,6 +210,7 @@ class cRoot
 
 	cCraftingRecipes * m_CraftingRecipes;
 	cFurnaceRecipe *   m_FurnaceRecipe;
+	cBrewingRecipes *  m_BrewingRecipes;
```

Comment:
Okay there are more lines, I think this should be done in own pull request. Or should I change only my line? :)